lib/charts/simple_bar_chart.dart

```diff
@@ -17,12 +17,15 @@ class SimpleBarChart extends StatelessWidget {
   }
 }
 
+/// Takes a Map and turns it into a list of [Tuple2].
 List<Tuple2<String, dynamic>> mapToTupleList(Map<String, dynamic> map){
   List<Tuple2<String, dynamic>> list = [];
   map.forEach((k,v) => list.add(Tuple2<String, dynamic>(k, v)));
   return list;
 }
 
+/// Takes the [labelValues] and [labelTitles] and creates a list of [charts.Series] from it.
+/// [labelTitles] will be on x-Axis, [labelValues] on y axis.
```

Comment:
[labelValues] on y axis. Is this right? I thought they only determine how large the bar becomes

Reply:
So..the y axis then? :D or rather the value on the y axis.

Reply:
The sentence implies that labelValues are next to the Y-axis, just like labels.

[labelTitles] annotate the X-axis. [labelValues] give the bars their size on the y-axis.

Reply:
```suggestion
/// [labelTitles] contain names of each bar, [labelValues] contain their sizes.
```